content/docs/get-started/aws/create-project.md

```diff
@@ -14,30 +14,33 @@ aliases: ["/docs/quickstart/aws/create-project/"]
 
 Let's get started with a new project in a new directory.
 
-{{< langchoose csharp >}}
+{{< chooser language "javascript,typescript,python,go,csharp" >}}
```

Comment:
Doesn't really apply to this line, but figured I'd mention it... We originally had the tabs laid out where JavaScript came before TypeScript. Then we deliberately decided to change it so that TypeScript came before JavaScript, given that that's the language most often used for NodeJS (and what we'd generally recommend).

Hopefully you don't mind switching the tabs rendering to show TypeScript first, which won't match the order of your language list and content throughout (which has JS first) 😬.

Reply:
Oh that's fine, yeah -- I wasn't sure which one it should be, but it's driven by the order defined in `chooser.tsx`, so it should be easy to do that.

Reply:
I've made the change to show TypeScript before JavaScript as well.